lib/megaphone_client/episode.rb

```diff
@@ -6,80 +6,102 @@ module MegaphoneClient
 
   # @author Jay Arella
   class Episode
-    class << self
 
-      # @return a MegaphoneClient
-      # @note This is used as a way to access top level attributes
-      # @example Accessing a network id
-      #   config.network_id #=> '{network id specified in initialization}'
+    # @return a MegaphoneClient::Episode instance
+    # @note This is used to initialize the podcast id and episode id when creating a new Episode instance
+    # @example Initialize a new instance of MegaphoneClient::Episode
+    #   MegaphoneClient::Episode.new("{podcast_id}", "{episode_id}") #=> #<MegaphoneClient::Episode @id="{episode_id}", @podcast_id="{podcast_id}">
 
-      def config
-        @config ||= MegaphoneClient
+    def initialize(podcast_id=nil, id=nil)
+      @id = id
+      @podcast_id = podcast_id
+    end
+
+    # @return a MegaphoneClient
+    # @note This is used as a way to access top level attributes
+    # @example Accessing a network id configuration
+    #   config.network_id #=> '{network id specified in initialization}'
+
+    def config
+      @config ||= MegaphoneClient
+    end
+
+    # @return a struct that represents the episode that was created
+    # @note If a @podcast_id, options[:title], and options[:pubdate] aren't given, it raises an error.
+    # @see MegaphoneClient#connection
+    # @example Create an episode
+    #   megaphone.podcast("12345").episode.create({
+    #     title: "title",
+    #     pubdate: "2020-06-01T14:54:02.690Z"
+    #   })
+    #   #=> A struct representing episode '12345' with title, "title", and scheduled to publish at June 1st, 2020
+
+    def create options={}
+      if !@podcast_id || !options || !options[:title] || !options[:pubdate]
```

Comment:
options[:title] is required, but where in this method does it get used?

Reply:
@nakedsushi It's required by the Megaphone API episode endpoint, so it's not necessarily used by the gem's API, but my thought is that it should probably prevent you from making a request unless it has the minimum requirements.